source/blender/makesdna/DNA_object_types.h

```diff
@@ -112,6 +112,21 @@ typedef struct LodLevel {
 	int obhysteresis;
 } LodLevel;
 
+typedef struct ObjectActivityCulling {
+	/* For game engine, values around active camera where physics or logic are suspended */
+	float physicsRadius;
+	float logicRadius;
+
+	int flags;
+	int pad;
+} ObjectActivityCulling;
+
+/* object activity flags */
+enum {
+	OB_ACTIVITY_PHYSICS = (1 << 0),
```

Comment:
@youle31 : I'm using a flag to disable some culling without setting the radius to zero in case the user want to test the culling without loose his radius settings.